setup.cfg

```diff
@@ -1,55 +1,3 @@
-[metadata]
-name = Django
-version = attr: django.__version__
-url = https://www.djangoproject.com/
-author = Django Software Foundation
-author_email = foundation@djangoproject.com
-description = A high-level Python web framework that encourages rapid development and clean, pragmatic design.
-long_description = file: README.rst
-license = BSD-3-Clause
-classifiers =
-    Development Status :: 2 - Pre-Alpha
-    Environment :: Web Environment
-    Framework :: Django
-    Intended Audience :: Developers
-    License :: OSI Approved :: BSD License
-    Operating System :: OS Independent
-    Programming Language :: Python
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.10
-    Programming Language :: Python :: 3.11
-    Programming Language :: Python :: 3.12
-    Topic :: Internet :: WWW/HTTP
-    Topic :: Internet :: WWW/HTTP :: Dynamic Content
-    Topic :: Internet :: WWW/HTTP :: WSGI
-    Topic :: Software Development :: Libraries :: Application Frameworks
-    Topic :: Software Development :: Libraries :: Python Modules
-project_urls =
-    Documentation = https://docs.djangoproject.com/
-    Release notes = https://docs.djangoproject.com/en/stable/releases/
-    Funding = https://www.djangoproject.com/fundraising/
-    Source = https://github.com/django/django
-    Tracker = https://code.djangoproject.com/
-
-[options]
-python_requires = >=3.10
-packages = find:
-include_package_data = true
-zip_safe = false
-install_requires =
-    asgiref >= 3.7.0
-    sqlparse >= 0.3.1
-    tzdata; sys_platform == 'win32'
-
-[options.entry_points]
-console_scripts =
-    django-admin = django.core.management:execute_from_command_line
-
-[options.extras_require]
-argon2 = argon2-cffi >= 19.1.0
-bcrypt = bcrypt
-
 [flake8]
```

Comment:
Optional: Wondering if we should rename `setup.cfg` to `.flake8`?

Pretty much every other tool now supports `pyproject.toml`, with `flake8` being an exception, and `setup.cfg` really belongs to `setuptools`.

Reply:
+1 to switching it. It may help avoid confusing other tools that look for data in `setup.cfg`.

Reply:
We could do, but I don't really like one tool using another tool's configuration file. This is why `pyproject.toml` is nice as it is intended to be tool-agnostic.

Also `tox` does have rudimentary support for `pyproject.toml`, but it's basically one setting with a large string blob of everything - so not worth merging that yet.

We could also keep `setup.cfg` and move `tox.ini` in there, which also works, but - as I said - `setup.cfg` really belongs to (legacy) `setuptools`.